constants/index.js

```diff
@@ -0,0 +1,70 @@
+const PATH_AVATARS = "/public/avatars";
+const AVATAR_GUEST = "guest";
+
+const DEFAULT_USER = "system";
+
+const SALT_ROUNDS = 12;
+const PEPPER_RANGE = 1000;
+
+const INVALID_EMAIL_ADDRESS = "Invalid Email Address!";
+const INVALID_PASSWORD = "Invalid Password!";
+const NAME_OR_EMAIL_TAKEN = "Invalid Name!";
+const INTERNAL_SERVER_ERROR = "Internal Server Error";
+const WRONG_PASSWORD = "Wrong Password!";
+
+const GRAPH_NOT_FOUND = "Graph not found!";
+const INVALID_GRAPH = "Invalid graph format!";
+const INTERNAL_ERROR = "Internal error occurred";
+
+const ERRORS = { GRAPH_NOT_FOUND, INVALID_GRAPH, INTERNAL_ERROR };
+
+const ERROR_CODES_MAP = new Map([
+  [GRAPH_NOT_FOUND, 404],
+  [INVALID_GRAPH, 400],
+  [INTERNAL_ERROR, 500],
+]);
+
+const CREATED = "Create";
+const READ_ONE = "Read One";
+const READ_MANY = "Read Many";
+const UPDATED = "Update";
+const DELETED = "Delete";
+
+const CRUD_OPS = {
+  CREATED,
+  READ_ONE,
+  READ_MANY,
+  UPDATED,
+  DELETED,
+};
+
```

Comment:
The more common approach is to do something like this : 

```js

const CRUD_OPS = {
  CREATED: "Create",
  READ_ONE: "Read One",
  READ_MANY: "Read Many",
  UPDATED: "Update",
  DELETED: "Delete",
};
```

You don't have to change it, its fine for now